src/features/user/ManageProjects/components/BasicDetails.tsx

```diff
@@ -114,23 +114,25 @@ export default function BasicDetails({
 
   const [projectCoords, setProjectCoords] = React.useState<number[]>([0, 0]);
 
-  const changeLat = (e: any) => {
-    if (e.target.value && e.target.value > -90 && e.target.value < 90) {
+  const changeLat = (e: React.ChangeEvent<HTMLInputElement>) => {
```

Comment:
perhaps import ChangeEvent from React instead of using React.ChangeEvent everywhere?